source/elements/oneMKL/source/domains/spblas/matrixinit.rst

```diff
@@ -24,10 +24,29 @@ The oneapi::mkl::sparse::init_matrix_handle function initializes the
 
    namespace oneapi::mkl::sparse {
 
-      void init_matrix_handle (oneapi::mkl::sparse::matrix_handle_t *handle);
+      void init_matrix_handle (oneapi::mkl::sparse::matrix_handle_t *p_handle);
 
    }
 
+
+.. container:: section
+
+    .. rubric:: Input parameter
+
+    p_handle
+       The address of the sparse::matrix_handle_t ``p_handle`` object to be initialized.
+       This initialization routine must only be called on an uninitialized matrix_handle_t object.
+
+
+.. container:: section
+
+    .. rubric:: Output parameters
+
+    p_handle
+       On return, the address is updated to point to a newly allocated and initialized matrix_handle_t object
+       that can be filled and used to perform sparse BLAS operations.
```

Comment:
I think it would be better to have `Input parameter` and `Output parameters` sections as well in `init_matrix_handle` routine, so I added them as well.

Reply:
awesome!  :)  I think it did a similar thing once upon a time to our oneMKL documentation as well

Reply:
Section name is just changed as `Input parameters`. :)